documentation/source/objectref/objects/groups.rst

```diff
@@ -24,16 +24,16 @@ It is important to understand that any changes to the returned group contents wi
 
 ::
 
-    group = font.groups["myGroup"]
+    group = list(font.groups["myGroup"])
     group.remove("A")
 
 If one wants to make a change to the group contents, one should do the following instead:
 
 ::
 
-    group = font.groups["myGroup"]
+    group = list(font.groups["myGroup"])
     group.remove("A")
-    font.groups["myGroup"] = group
+    font.groups["myGroup"] = tuple(group)
```

Comment:
I think the item setter does accept lists, so it should not be necessary to convert back to tuple here.

Reply:
Ah, good to know. Here is the update: 
https://github.com/robotools/fontParts/pull/501/commits/1d12906d4e01d43415681f14156abe6fada896b3